### ISSUE TITLE
Slack post may have failed. Response: No hooks

### ISSUE THREAD

**Issue by Bilalaybar** [jenkinsci/slack-plugin#323]
Hi there ,

I've got multiple Jenkins and one slack team account,

I'm able to send a notification to my slack channel from 2 instances, however from my other Jenkins instance I can't.Configurations are the same.

![image](https://cloud.githubusercontent.com/assets/6894861/26724859/3b664724-47a4-11e7-87eb-3ee41b59bbeb.png)

Here are the logs that I hit Test Connection button: 
![image](https://cloud.githubusercontent.com/assets/6894861/26724890/5ad36254-47a4-11e7-9720-d42aff939bc9.png)

Haz 02, 2017 2:48:30 PM WARNING jenkins.plugins.slack.StandardSlackService publish
Slack post may have failed. Response: No hooks
Haz 02, 2017 2:48:30 PM WARNING jenkins.plugins.slack.StandardSlackService publish
Response Code: 404

PS: Plugin versions and Jenkins Versions are same in 3 Jenkins machines.

Thank u for ur help,

**Comment by gbyukg** [comment]
The same error messages in my Jenkins server.

**Comment by joaquin386** [comment]
I get an error 301:
WARNING: Response Code: 301
Nov 24, 2017 3:37:27 PM jenkins.plugins.slack.StandardSlackService publish
WARNING: Slack post may have failed. Response:

**Comment by a-drees** [comment]
I am having the same issue.  Runnins Jenkins in master/node configuration with the master as a docker container.

Has there been any progress on this at all?

**Comment by boianmihailov** [comment]
I am having the same issue, but trying to send  multiple slack notifications to different organisations. 
If theres settings saved in global configration, token or secret text token id, second notification always fails with error "Slack post may have failed. Response: No hooks"

**Comment by matthewlowry** [comment]
I know this is an old issue but just adding my experience:

I was getting this same error in the logs and no messages being sent, but it was a simple mistake on my part.

I had done the global configuration for the plugin, and clicked the "Test connection" button, and the test message worked as intended. But then when I ran my pipeline jobs no messages were being sent. Turned out I had simply failed to save the global configuration. In the pipeline jobs the error messages didn't say something like "Hey wait you've got no configuration", so it took me a while to work out what was going on. Oops.🥇 for me.
